experimental/dds/tree2/.dependency-cruiser-known-violations.json

```diff
@@ -1,111 +1,111 @@
 [
```

Comment:
No actual changes here. I've extracted un-applying of prettier into a separate PR: #18439